Account/FirefoxAccountConfiguration.swift

```diff
@@ -91,7 +94,7 @@ public struct StableDevFirefoxAccountConfiguration: FirefoxAccountConfiguration
 }
 
 public struct StageFirefoxAccountConfiguration: FirefoxAccountConfiguration {
-    public init() {
+    public init(prefs: Prefs? = nil) {
```

Comment:
This change isn't needed if you remove the constructor from the `FirefoxAccountConfiguration`.